spring-cloud-task-core/src/main/java/org/springframework/cloud/task/repository/dao/JdbcTaskExecutionDao.java

```diff
@@ -165,16 +169,29 @@ public TaskExecution createTaskExecution(String taskName,
 	}
 
 	@Override
-	public TaskExecution startTaskExecution(long executionId, String taskName, Date startTime, List<String> arguments,
+	public TaskExecution startTaskExecution(long executionId, String taskName,
+			Date startTime, List<String> arguments,
 			String externalExecutionId) {
-		TaskExecution taskExecution = new TaskExecution(executionId, null, taskName,
-				startTime, null, null, arguments, null, externalExecutionId);
+		TaskExecution taskExecution = new TaskExecution(executionId, null,
+				taskName, startTime, null, null, arguments, null,
+				externalExecutionId);
+
+		Object[] queryParameters = new Object[]{ startTime, taskName,
```

Comment:
Very small nit pick here.  For efficiency, it would be better to create this array in an `else` of the `if` statement below (instead of always creating two arrays.  Same goes for the `argTypes` arrays as well.